frappe/public/js/frappe/form/workflow.js

```diff
@@ -112,9 +112,6 @@ frappe.ui.form.States = Class.extend({
 					} else if(new_docstatus==2 && me.frm.doc.docstatus==1) {
 						me.frm.savecancel(null, success, on_error);
 					} else {
-						frappe.msgprint(__("Document Status transition from ") + me.frm.doc.docstatus + " "
```

Comment:
why remove this?

Reply:
That seemed like duplicate msgprint statement, so asked Rohit it was okay to remove it or so